core-feature-pack/common/src/main/resources/content/bin/standalone.sh

```diff
@@ -309,7 +309,7 @@ fi
 SECURITY_MANAGER_SET=`echo $JAVA_OPTS | $GREP "java\.security\.manager"`
 if [ "x$SECURITY_MANAGER_SET" != "x" ]; then
     SECURITY_MANAGER_SET_TO_ALLOW=`echo $JAVA_OPTS | $GREP "java\.security\.manager=allow"`
-    if [ "x$SECURITY_MANAGER_SET_TO_ALLOW" == "x" ]; then
+    if [ "x$SECURITY_MANAGER_SET_TO_ALLOW" = "x" ]; then
```

Comment:
Hi @rosiecki thanks for the contribution but this is a valid string comparison for any POSIX shell; you will also see it used widely in this script. 

In any case, just for your info, if you detect a bug the preferred approach is to report it by creating a Jira describing how to reproduce it.

I'm going to close this pull request, but please, feel free to comment here if you have any questions or notes about this. Thanks!

Reply:
> you will also see it used widely in this script


let me check:

``
fgrep "==" wildfly-26.1.0.Beta1/bin/standalone.sh
``

the result is only one:

``
    if [ "x$SECURITY_MANAGER_SET_TO_ALLOW" == "x" ]; then
echo "========================================================================="
echo "========================================================================="
``

Reply:
Ups, I'm sorry @rosiecki , I got confused with the change, you are removing the double == which should be fine since the script uses a single = in general. 

I'm curious since == and = are both valid operators for a POSIX compatible shell, could you describe on which OS are you hitting the issue?

Reply:
I found the issue on Ubuntu 18.04.5 LTS.

Reply:
@rosiecki thanks, now I got it, and == is not a POSIX compatible operator that causes; I made two mistakes reviewing this.

I've created this Jira to track it: https://issues.redhat.com/browse/WFCORE-5871

Would you mind changing your commit message to include the Jira number? See https://docs.wildfly.org/26/Hacking_On_WildFly.html#contributing_commit_push for getting more info.

Reply:
In addition to the above and due to the changes you are doing here, we require that all contributions be made under the terms of the MIT License, http://www.opensource.org/licenses/mit-license.php. Do you agree to these terms?

Reply:
> In addition to the above and due to the changes you are doing here, we require that all contributions be made under the terms of the MIT License, http://www.opensource.org/licenses/mit-license.php. Do you agree to these terms?

I agree with the terms.

Reply:
Thanks, @rosiecki, we need you to follow https://docs.wildfly.org/26/Hacking_On_WildFly.html#contributing_commit_push instructions and change the commit message to include the Jira issue, after that, we will enable CI on this PR, and then we will ready for merging it :)